.eslintrc.js

```diff
@@ -1,12 +1,6 @@
 module.exports = {
   parser: '@babel/eslint-parser',
   settings: {
-    'import/resolver': {
```

Comment:
This is solely to service the TDP test? How about instead of this, we revert this change and add this to `.eslintrc.js` to make it clearer this is only for TDP:

```
        moduleDirectory: [
          'node_modules',
          // TDP module path is for resolving js-cookie in TDP tests.
          'cfgov/unprocessed/apps/teachers-digital-platform/node_modules',
        ],
```

Reply:
cc @wpears